src/restapi/mm_routes.ts

```diff
@@ -19,21 +19,27 @@ const router = express.Router();
 router.get(Routes.App.ManifestPath, fManifest);
 router.post(Routes.App.BindingsPath, fBindings);
 router.get(Routes.App.OAuthCompletePath, fComplete);
-
 router.post(Routes.App.InstallPath, fInstall);
 
-// Location bound calls
-router.post(Routes.App.BindingPathConnect, fConnect);
-router.post(Routes.App.BindingPathDisconnect, fDisconnect);
-router.post(Routes.App.BindingPathOpenCreateTicketForm, fOpenCreateTicketForm);
-router.post(Routes.App.BindingPathOpenSubscriptionsForm, fOpenSubscriptionsForm);
-router.post(Routes.App.BindingPathOpenZendeskConfigForm, fOpenZendeskConfigForm);
-router.post(Routes.App.BindingPathHelp, fHelp);
-
-// Callable routes, not bound to a location
-router.post(Routes.App.CallPathSubmitOrUpdateCreateTicketForm, fSubmitOrUpdateCreateTicketForm);
-router.post(Routes.App.CallPathSubmitOrUpdateSubscriptionForm, fSubmitOrUpdateSubscriptionsForm);
-router.post(Routes.App.CallPathSubmitOrUpdateZendeskConfigForm, fSubmitOrUpdateZendeskConfigForm);
+// formless calls
+router.post(Routes.App.BindingPathConnectSubmit, fConnect);
```

Comment:
1/5 s/`BindingPathConnectSubmit `/`connect/submit` (we don't use the const anywhere and no need to search for it - inline-ing it should be ok) + the others.

Reply:
Removed all `/submit` and `/form` endpoints from `constants.ts` file. These are routes that only need to be defined one time. The routes that remain in
`utils/constants.ts` `Routes.App` are call paths that can be used by multiple locations